requirements.txt

```diff
@@ -1,3 +1,4 @@
-jupyter-book
-matplotlib
-numpy
+git+https://github.com/executablebooks/jupyter-book
+altair_saver
```

Comment:
Can we just delete this file now? Our build environment is entirely in docker now

Reply:
Yup I think so